packages/eservice-template-process/test/api/patchUpdateDraftTemplateVersion.test.ts

```diff
@@ -0,0 +1,265 @@
+      dailyCallsTotal: 200,
+      agreementApprovalPolicy: "AUTOMATIC",
+      description: "new description",
+      attributes: {
+        certified: [
+          [{ id: getMockAttribute().id, explicitAttributeVerification: false }],
+        ],
+        declared: [
+          [{ id: getMockAttribute().id, explicitAttributeVerification: false }],
+        ],
+        verified: [
+          [{ id: getMockAttribute().id, explicitAttributeVerification: false }],
+        ],
+      },
+    };
+
+  eserviceTemplateService.patchUpdateDraftTemplateVersion = vi
+    .fn()
+    .mockResolvedValue(serviceResponse);
+
+  const makeRequest = async (
+    token: string,
+    templateId: EServiceTemplateId,
+    versionId: EServiceTemplateVersionId,
+    body: eserviceTemplateApi.PatchUpdateEServiceTemplateVersionSeed = versionSeed
+  ) =>
+    request(api)
+      .patch(`/templates/${templateId}/versions/${versionId}`)
+      .set("Authorization", `Bearer ${token}`)
+      .set("X-Correlation-Id", generateId())
+      .send(body);
+
+  const authorizedRoles: AuthRole[] = [authRole.M2M_ADMIN_ROLE];
+  it.each(authorizedRoles)(
+    "Should return 200 for user with role %s",
+    async (role) => {
+      const token = generateToken(role);
+      const res = await makeRequest(
+        token,
+        mockEServiceTemplate.id,
+        templateVersion.id
+      );
+      expect(res.status).toBe(200);
+      expect(res.body).toEqual(apiEserviceTemplate);
+      expect(res.headers["x-metadata-version"]).toBe(
+        serviceResponse.metadata.version.toString()
+      );
+    }
+  );
+
+  it.each([
+    {},
+    {
+      description: "new description",
+    },
+    {
+      description: "new description",
+      voucherLifespan: 1000,
+    },
+    {
+      description: "new description",
+      voucherLifespan: 1000,
+      dailyCallsPerConsumer: 100,
+    },
+    {
+      description: "new description",
+      voucherLifespan: 1000,
+      dailyCallsPerConsumer: 100,
+      dailyCallsTotal: 200,
+    },
+    {
+      description: "new description",
+      voucherLifespan: 1000,
+      dailyCallsPerConsumer: 100,
+      dailyCallsTotal: 200,
+      agreementApprovalPolicy: "AUTOMATIC",
+    },
+    {
+      description: "new description",
+      voucherLifespan: 1000,
+      dailyCallsPerConsumer: 100,
+      dailyCallsTotal: 200,
+      agreementApprovalPolicy: "AUTOMATIC",
+    },
+    {
+      attributes: {
+        certified: [],
+        declared: [
+          [{ id: getMockAttribute().id, explicitAttributeVerification: false }],
+        ],
+      },
+    },
+    {
+      attributes: {
+        verified: [
+          [
+            { id: getMockAttribute().id, explicitAttributeVerification: false },
+            { id: getMockAttribute().id, explicitAttributeVerification: false },
+          ],
+        ],
+      },
+    },
+  ] as eserviceTemplateApi.PatchUpdateEServiceTemplateVersionSeed[])(
+    "Should return 200 with partial seed (seed #%#)",
```

Comment:
Also, add cases with nullable fields set to `null` above (see [here](https://vscode.dev/github/pagopa/interop-be-monorepo#L116) for example)

```suggestion
    "Should return 200 with partial seed and nullable fields (seed #%#)",
```